client/a8c-for-agencies/sections/signup/controller.tsx

```diff
@@ -6,15 +6,35 @@ import {
 	A4A_SIGNUP_FINISH_LINK,
 	A4A_SIGNUP_LINK,
 } from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import { useSelector } from 'calypso/state';
+import {
+	getActiveAgency,
+	hasFetchedAgency,
+	isFetchingAgency,
+} from 'calypso/state/a8c-for-agencies/agency/selectors';
 import { hideMasterbar } from 'calypso/state/ui/actions';
 import AgencySignUp from './primary/agency-signup';
 import AgencySignupFinish from './primary/agency-signup-finish';
 
+const PageViewTrackerWrapper = ( { path }: { path: string } ) => {
+	const agency = useSelector( getActiveAgency );
+	const hasFetched = useSelector( hasFetchedAgency );
+	const isFetching = useSelector( isFetchingAgency );
+
+	const shouldRenderTracker = hasFetched && ! isFetching && ! agency;
```

Comment:
I thought this would be enough to determine if we've fetched the agency and whether it returned an agency or null. But it seems `hasFetched` can be true and `agency` can continue to change (ie from null to an agency object). Not yet sure why this wouldn't work. Any feedback is welcome